testing/participant/ParticipantShape.ttl

```diff
@@ -70,6 +69,44 @@ shapes:ParticipantShape
 		sh:severity sh:Violation ;
 		sh:message "<https://raw.githubusercontent.com/International-Data-Spaces-Association/InformationModel/master/testing/participant/ParticipantShape.ttl> (ParticipantShape): An ids:participantCertification property must not have more than one point from an ids:Participant to an ids:ParticipantCertification."@en ;
 	];
+	
+	sh:property [
+		a sh:PropertyShape ;
+		sh:path ids:hasBusinessIdentifier ;
+		sh:class ids:BusinessIdentifier ;
+		sh:severity sh:Violation ;
+		sh:message "<https://raw.githubusercontent.com/International-Data-Spaces-Association/InformationModel/master/testing/participant/ParticipantShape.ttl> (ParticipantShape): An ids:hasBusinessIdentifier property must point from an ids:Participant to an ids:BusinessIdentifier."@en ;
+	];
+
+	sh:property [
+		a sh:PropertyShape ;
+		sh:path ids:vatID ;
+		sh:datatype xsd:string ;
+		sh:severity sh:Violation ;
+		#sh:minCount 1 ;
```

Comment:
We should either remove this commented-out line, or add a proper comment to explain possible future development. Please also see the two following properties below.